src/NzbDrone.Core/Localization/Core/en.json

```diff
@@ -635,6 +635,7 @@
   "EpisodeRequested": "Episode Requested",
   "EpisodeSearchResultsLoadError": "Unable to load results for this episode search. Try again later",
   "EpisodeTitle": "Episode Title",
+  "EpisodeTitleFootNote": "Episode Title/CleanTitle support a `:MAX_LENGTH` suffix allowing you to truncate the episode title. Ellipsis (`...`) will be added when truncating. The length of the truncated text (including ellipsis) will be equal to the specificied max length. For example `{Episode Title:30}`.",
```

Comment:
The information feels a little repetitive here, I don't think the token name needs to be repeated and can probably be cut down to a single sentence with examples.

Truncating from the beginning is also supported with `:-30` and the actual truncation is done on the number of bytes (which is mostly an issue when truncating due to file length limits).